qiskit/ignis/verification/accreditation/fitters.py

```diff
@@ -93,7 +94,8 @@ def bound_variation_distance(self, theta):
                 theta (float): number between 0 and 1
         """
         if self.N_acc == 0:
-            sys.exit()
+            QiskitError("ERROR: Variation distance requires"
+                        + "at least one accepted run")
```

Comment:
You don't need a `+` when the strings are between parentheses. This is fine:
```suggestion
                        "at least one accepted run")
```